pkg/apis/externaldns/version.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2025 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package externaldns
+
+import (
+	"fmt"
+	"runtime"
+)
+
+const (
+	bannerTemplate = `Version=%s, GoVersion=%s, GitCommitShort=%s, Platform=%s, UserAgent=%s`
```

Comment:
If UserAgent contains version, maybe we can remove `Version=%s` and avoid displaying same info twice ?

Reply:
Changed. We could revisit this if there is a requirement